### ISSUE TITLE
web server txt

### ISSUE THREAD

**Issue by bobokev** [NoahCardoza/HE-REvived-Issue-Tracker#34]
always found that boring just to add txt would it be possible to add bbcode or something  to it

**Comment by NoahCardoza** [comment]
I’d rather use markdown but I like the idea, it would be pretty easy to implement.

**Comment by Johannes2306** [comment]
I agree with @NoahCardoza, markdown would be better as it has more customizability...
Or maybe have some version of webserver which allows you to fully customize your VPCs landing page with HTML?
